src/aleph/http/http2.clj

```diff
@@ -423,7 +423,10 @@
                default-value
                (if (== 1 (.size vs))
                  (.toString (.get vs 0))
-                 (str/join "," vs))))))))
+                 (str/join "," vs)))))))
+  common/HeaderMap
+  (get-header-values [_ k]
+                     (.getAll headers ^String k)))
```

Comment:
You know... I'm not sure I knew this would work.

I've known for a while that you can sneak extra methods into Clojure-generated Java classes, even if not descirbed by a protocol/interface (iirc), but I kind of assumed `def-map-type` was more restricted. Probably should have taken a closer look at it.